app/src/main/java/com/github/sdpteam15/polyevents/EventActivity.kt

```diff
@@ -1,21 +1,27 @@
 package com.github.sdpteam15.polyevents
 
 import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.Button
 import android.widget.ImageView
 import android.widget.TextView
 import androidx.appcompat.app.AppCompatActivity
 import com.github.sdpteam15.polyevents.database.Database.currentDatabase
 import com.github.sdpteam15.polyevents.database.observe.Observable
+import com.github.sdpteam15.polyevents.exceptions.MaxAttendeesException
 import com.github.sdpteam15.polyevents.fragments.EXTRA_EVENT_ID
-import com.github.sdpteam15.polyevents.helper.HelperFunctions
+import com.github.sdpteam15.polyevents.helper.HelperFunctions.showToast
 import com.github.sdpteam15.polyevents.model.Event
 
+const val TAG = "EventActivity"
```

Comment:
If this constant will be used in another class, try to put it in a companion object rather than "just outside the class bound"

Reply:
Yes you're right! Thanks